autoreduce_qp/queue_processor/reduction/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/autoreduction/autoreduce:latest
+COPY . .
+RUN export DEBIAN_FRONTEND=noninteractive
+ARG DEBIAN_FRONTEND=noninteractive
+RUN python3 -m pip install autoreduce-qp==22.0.0.dev52 --extra-index-url https://api.packagr.app/public
+RUN python3 -m pip install pytest
+RUN python3 -m pip install parameterized
+USER isisautoreduce
```

Comment:
Can this dockerfile be moved to https://github.com/autoreduction/autoreduce-containers or a new repo just for these runner images? 

After this PR is merged I guess we want to stop using the extra index `https://api.packagr.app/public` and move to the production package